starknet_py/net/schemas/rpc/contract.py

```diff
@@ -181,3 +195,28 @@ class CasmClassSchema(Schema):
     @post_load
     def make_dataclass(self, data, **kwargs) -> CasmClass:
         return CasmClass(**data)
+
+
+class DeprecatedCompiledContractSchema(ContractClassSchema):
```

Comment:
Let's move this class to have it right below the `DeprecatedContractClassSchema`.